samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb

```diff
@@ -587,7 +589,132 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Clean up"
+    "## Customize a step in Python2\n",
```

Comment:
not important in this PR. generally, i feel it's a bad practice to demonstrate a bunch of small features in a big sample. It's very hard for user to discover this part. We should consider to create small demonstrate samples for a specific feature. For example:  python2 lambda in this case.

Reply:
Agreed. Let me add a comment for a TODO item.